dev/PowerNotifications/PowerNotifications.idl

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// REVIEW: We're not doing doing any versioning within IDL/WinMDs, right?
```

Comment:
Outstanding review comment. I left this here because I wasn't sure if we needed to tag version info in IDL on anything. Not actually 100% sure of the answer to it. @jonwis, are we doing any WinRT IDL versioning in Reunion?

Reply:
Per a recent conversation we intend to do contracts in IDL but some artifacts aren't defined yet

@jonwis @MikeHillberg we talked about the apicontract plan but I don't recall who was tagged to handle implementation. Who should Brian follow up with?